log.go

```diff
@@ -54,13 +55,15 @@ var (
 	walletLog = backendLog.Logger("WALL")
 	winLog    = backendLog.Logger("UI")
 	dlwlLog   = backendLog.Logger("DLWL")
+	pageLog   = backendLog.Logger("PAGE")
```

Comment:
Rebase with master, this change has already been made https://github.com/planetdecred/godcr/blob/master/log.go#L58